lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
Understood, but I'm curious about the mechanics. Sounds like it will fail regardless of whether it's minor or critical? How would/could we filter on minor vs critical when running tests?